plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -94,84 +88,87 @@ class SentryPlugin : Plugin<Project> {
-
-                    if (isMinifyEnabled) {
-                        // Setup the task to generate a UUID asset file
-                        val uuidOutputDirectory = project.file(
-                            "build${sep}generated${sep}assets${sep}sentry${sep}${variant.name}"
-                        )
-                        val generateUuidTask = project.tasks.register(
-                            "generateSentryProguardUuid$taskSuffix",
-                            SentryGenerateProguardUuidTask::class.java
-                        ) {
-                            it.outputDirectory.set(uuidOutputDirectory)
-                        }
-                        variant.mergeAssetsProvider.configure { it.dependsOn(generateUuidTask) }
-
-                        // Setup the task that uploads the proguard mapping and UUIDs
-                        val uploadSentryProguardMappingsTask = project.tasks.register(
-                            "uploadSentryProguardMappings$taskSuffix",
-                            SentryUploadProguardMappingsTask::class.java
-                        ) {
-                            it.dependsOn(generateUuidTask)
-                            it.workingDir(project.rootDir)
-                            it.cliExecutable.set(cliExecutable)
-                            it.sentryProperties.set(
-                                sentryProperties?.let { file -> project.file(file) }
-                            )
-                            it.uuidDirectory.set(uuidOutputDirectory)
-                            mappingFile?.let { mapFile ->
-                                it.mappingsFile.set(mapFile)
-                            }
-                            it.autoUpload.set(extension.autoUpload.get())
-                            it.sentryOrganization.set(sentryOrgParameter)
-                            it.sentryProject.set(sentryProjectParameter)
-                        }
-                        androidExtension.sourceSets.getByName(variant.name).assets.srcDir(
-                            uuidOutputDirectory
-                        )
-
-                        // and run before dex transformation. If we managed to find the dex task
-                        // we set ourselves as dependency, otherwise we just hack ourselves into
-                        // the proguard task's doLast.
-                        dexTask?.dependsOn(uploadSentryProguardMappingsTask)
-                        transformerTask?.finalizedBy(uploadSentryProguardMappingsTask)
-
-                        // To include proguard uuid file into aab, run before bundle task.
-                        preBundleTask?.dependsOn(uploadSentryProguardMappingsTask)
+                var taskSuffix = variant.name
+                // capitalizeUS
+                taskSuffix = if (taskSuffix.isEmpty()) {
+                    ""
+                } else {
+                    taskSuffix.substring(0, 1).toUpperCase(Locale.US) + taskSuffix.substring(1)
+                }
```

Comment:
Why are you doing this? Plus this should potentially be exported to a function and unit tested.

Reply:
Gotcha 👍 
You can also reword it with:

```suggestion
                var taskSuffix = variant.name
                // capitalizeUS
                taskSuffix = if (taskSuffix.isEmpty()) {
                    ""
                } else {
                    taskSuffix.first().toUpperCase(Locale.US) + taskSuffix.drop(1)
                }
```

Reply:
extracted and added test to it

Reply:
`Char` does not have a .toUpperCase(locale) so cant use it if I want to `toUpperCase(Locale.US)`

Reply:
`drop` calls `substring` so its not needed, just one more method call